### PR TITLE
S260106 : Upgrade toolkit to work with Jetty12 

### PR DESCRIPTION
**Pull request by kworam-rally**
upgrade to work correctly with Jetty12

[S260106](https://rally1.rallydev.com/#/?detail=/userstory/735153164983&fdp=true): Upgrade RallyRestToolkitForJava to support Jetty12

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -28,6 +32,10 @@
             <id>krmorse</id>
```
Comment (broadcom-tony) [inline pom.xml:32]: delete this whole developers section... I don't think we want people to think we actually actively develop this anymore

Comment (kworam-rally) [reply pom.xml]: done

**Review by klehman-rally — COMMENTED** [review]
I note that the language level changed from JDK_1_5 to JDK_1_8.
What effect would there be to changing the language level to a version that is more current?  Would there be any benefit to that?

**Comment by kworam-rally** [comment]
> I note that the language level changed from JDK_1_5 to JDK_1_8. What effect would there be to changing the language level to a version that is more current? Would there be any benefit to that?

I wanted to make the minimal change to get the toolkit fixed.  It seems to me there won't be much benefit to upgrading to a more recent JDK and it might break some other code somewhere in the toolkit

**Pull request merged**